lib/Doctrine/DBAL/Logging/LoggerChain.php

```diff
@@ -7,12 +7,22 @@
  */
 class LoggerChain implements SQLLogger
 {
-    /** @var SQLLogger[] */
+    /** @var iterable<SQLLogger> */
     private $loggers = [];
 
+    /**
+     * @param SQLLogger[] $loggers
```

Comment:
Please use the same type as on the property. Preferrable, `array<int, SQLLogger>`.

Reply:
we don't care about keys so type shouldn't specify them